conf/nginx.conf

```diff
@@ -1,5 +1,5 @@
 location PATHTOCHANGE/static/ {
-    alias   /opt/yunohost/ihatemoney/venv/lib/python3.4/site-packages/ihatemoney/static/;
+    alias   APPHOME/static/;
```

Comment:
Is that backward compatible ? If not, I think there is something to set in manifest to tell that it is only compatible ynh >=3.

Reply:
I think not, I haven't worked on the update :s

Reply:
@klorydryk OK :) do you plan to in this PR ?